CryptoPrices/Sources/Styleguide/Sources/Styleguide/Extensions/Font+Inter.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  Font+Inter.swift
+//  Styleguide
+//
+//  Created by Minh Pham on 06/12/2022.
+//
+
+import SwiftUI
+import Foundation
+
+public extension Font {
+    
+    enum Inter: CaseIterable {
+
+        case regular
+        case medium
+        case bold
+
+        public func font(for style: UIFont.TextStyle) -> Font {
+            let fontSize = fontSize(style: style)
+            let finalFont = Font.custom(fontName(), fixedSize: fontSize)
+            return finalFont
+        }
+
+        public static func registerFonts() {
+            Inter.allCases.forEach {
+                registerFont(bundle: .module, fontName: $0.fontName(), fontExtension: "ttf")
+            }
+        }
+
+        func fontName() -> String {
+            switch self {
+            case .regular: return "Inter-Regular" // Weight: 400
+            case .medium: return "Inter-Medium" // Weight: 500
+            case .bold: return "Inter-Bold" // Weight: 700
+            }
+        }
+
+        // swiftlint:disable:next cyclomatic_complexity
+        func fontSize(style: UIFont.TextStyle) -> CGFloat {
+            switch style {
+            case .largeTitle: return 34.0
+            case .title1: return 28.0
```

Comment:
Since we're using `SwiftUI` rather than `UIKit`, I'd prefer to use `Font.TextStyle` than `UIFont.TextStyle`, although there's not much difference here (`Font.TextStyle` might not have `title1` and `caption1` which is not important I think 🙏 )